spec/support/shared_examples/behaves_like_invitable.rb

```diff
@@ -49,12 +49,12 @@
 
     it 'rejects banned attending students' do
       invitation_to_banned_student = Fabricate(invitation_type,
-                                        member: Fabricate(:banned_member),
-                                        role: 'Student',
-                                        invitable_type => invitable,
-                                        attending: true)
+                                               member: Fabricate(:banned_member),
+                                               role: 'Student',
+                                               invitable_type => invitable,
+                                               attending: true)
 
-                                        # byebug
+      # byebug
```

Comment:
@notapatch I think this could be deleted, I doubt we need this here.

Reply:
great... removed